### PR TITLE
Tali improvements

### PR DESCRIPTION
**Pull request by therzka**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,8 @@ $ pip install .
 
 ## Example usage
 
+access help by typing `$ transcriptic --help` or `$ transcriptic [COMMAND] --help`
```
Comment (nornagon) [inline README.md]: capital letter & final period pls

File: README.md [hunk 2]
```diff
@@ -36,10 +38,17 @@ $ python my_protocol.py | transcriptic analyze
...
 ##### Submit a protocol to Transcriptic
 ```
-$ python my_protocol.py | transcriptic submit --project sequencing --title "Sequencing run"
+$ python my_protocol.py | transcriptic submit --project "sequencing" --title "Sequencing run"
 Run created: https://secure.transcriptic.com/cambridge/sequencing/r1xa043277aekj
 ```
 
+##### Run a protocol by passing it a .json file with parameters
+```
+$ transcriptic run my_protocol.py protocol_config.json
```
Comment (nornagon) [inline README.md]: pretty sure this needs to be `transcriptic run MyProtocol protocol_config.json`.

File: transcriptic.py [hunk 3]
```diff
@@ -76,12 +77,12 @@ def cli(ctx, config, organization):
 @click.argument('file', default='-')
 @click.option('--project', '-p',
               metavar='PROJECT_ID',
-              help='Project to submit the run to',
-              required=True)
-@click.option('--title', '-t', help='Title the run')
-@click.option('--test', help="submit this run in test mode.", is_flag=True)
+              required=True, help='Project to submit the run to')
+@click.option('--title', '-t', help='Optional title of your run')
+@click.option('--test', help='submit this run in test mode.', is_flag=True)
```
Comment (nornagon) [inline transcriptic.py]: be consistent with punctuation.

**Comment by therzka** [comment]
@nornagon fixed

**Pull request merged**